package.json

```diff
@@ -49,6 +49,8 @@
     "react-app-polyfill": "^1.0.2",
     "react-dev-utils": "^9.0.3",
     "react-dom": "^16.9.0",
+    "react-router": "^5.0.1",
```

Comment:
tu się podzielę refleksją dotyczącą `^`
dla `^5.0.1` znak ten pozwala na instalację wersji `5.1.0`

teoretycznie, zgodnie z założeniami semver (semantyczne wersjonowanie), podbicie z `5.0.1` na `5.1.0` nie powinno nic popsuć, bo jakakolwiek breaking change w danym pakiecie powinno skutkować podbiciem numeru major, czyli wskoczeniem na `6.0.0`

praktycznie, przynajmniej w projektach, w których ja byłem, stosowaliśmy restrykcyjne podejście polegające na lockowaniu wersji pakietów bez `~` czy `^`, jeśli ma być `5.0.1` to ma być `5.0.1`
wtedy projekt jest bardziej przewidywalny/stabilny, bo jednak jakiś bug może się wkraść w wersji `5.1.0` jakiegoś pakietu i apka się posypie

mówię to jako ogólną refleksję, nie postuluję takiej zmiany w naszym projekcie

Reply:
https://semver.org/
```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.
```

Reply:
👍